operator/src/main/resources/scripts/livenessProbe.sh

```diff
@@ -34,4 +34,8 @@ if [ -f ${STATEFILE} ] && [ `grep -c "FAILED_NOT_RESTARTABLE" ${STATEFILE}` -eq
   trace "Error: WebLogic Server state is FAILED_NOT_RESTARTABLE."
   exit $RETVAL
 fi
+if [ -f ${STATEFILE} ] && [ `grep -c "SHUTDOWN" ${STATEFILE}` -eq 1 ]; then
+  trace "Error: WebLogic Server state is SHUTDOWN."
+  exit $RETVAL
+fi
```

Comment:
Oh, I see... You did include this.

Reply:
@markxnelson, I've been thinking about this...  It does make sense that the operator should detect servers that have the "wrong" status, such as being shutdown when we think they are running, since this could sometimes be a mistake.  However, I could also see wanting to intentionally "exec bash" into a server to diagnose or correct something and not want to have the liveness probe restart the server until you've completed your analysis.  I'd be okay with a change where there was an easy way to temporarily make the liveness probe return success (different than the behavior that completely mocks out the liveness probe).  @tbarnes-us, what do you think?

Reply:
>  Ryan:  I'd be okay with a change where there was an easy way to temporarily make the liveness probe return success (different than the behavior that completely mocks out the liveness probe).

@rjeberhard The good fairy dropped by and granted your wish, although I'm not sure it qualifies as easy:  there's an internal debugging option to temporarily force the liveness probe return success.  

The way it works is that the server pods optionally mount a configmap named "DOMAIN_UID-weblogic-domain-debug-cm" and then the liveness probe and start server script both check for the file "/weblogic-operator/debug/livenessProbeSuccessOverride". 

An end user can enable the option by doing something like this: 

```
  # The following dynamically enables a debugging option that forces 
  # wl pod liveness probes and start server scripts to always return 
  # success/running even if the wl server crashes.  

  kubectl -n ${NAMESPACE} create configmap \
              ${DOMAIN_UID}-weblogic-domain-debug-cm \
              --from-literal=livenessProbeSuccessOverride=whatever

``` 

We originally agreed to make this option internal use only, but maybe we should just go ahead and officially document and support it.